OpenTaiko/src/Common/ImGuiDebugWindow.cs

```diff
@@ -366,17 +364,126 @@ private static void Textures() {
 					}
 					ImGui.EndCombo();
 				}
+				if (OpenTaiko.r現在のステージ.eStageID != CStage.EStage.StartUp)
+					CTextureListPopup(OpenTaiko.Tx.listTexture, "Show listTexture", "TEXTURE_ALL");
+				else
+					ImGui.TextDisabled("To prevent crash during enumeration,\nyou can not view the texture list during StartUp stage.");
+
+				currentStageMemoryUsage = 0;
 
-				if (ImGui.TreeNodeEx("Show All Textures###TEXTURE_SHOWALL")) {
-					int index = 0;
-					if (OpenTaiko.r現在のステージ.eStageID == CStage.EStage.StartUp)
-						ImGui.TextDisabled("To prevent crash during enumeration,\nyou can not view the texture list during StartUp stage.");
-					else
-						foreach (CTexture tex in OpenTaiko.Tx.listTexture) {
-							CTexturePopup(tex, $"#{index} (Pointer: {(tex != null ? tex.Pointer : "null")})###TEXTURE_SHOW_POPUP_{index++}");
+				#region Script.lua Memory Usage
+				switch (OpenTaiko.r現在のステージ.eStageID) {
+					#region Game
+					case CStage.EStage.Game:
+						if (ImGui.TreeNodeEx("Loaded Script.lua Textures###TEXTURE_LUA")) {
+
+							currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actBackground.UpScript,
+								"Up Background", "TEXTURE_LUA_UPBG");
+							currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actBackground.DownScript,
+								"Down Background", "TEXTURE_LUA_DOWNBG");
+							currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actMob.MobScript,
+								"Mob", "TEXTURE_LUA_MOB");
+
+							#region Endings
+							switch ((Difficulty)OpenTaiko.stageSongSelect.nChoosenSongDifficulty[0]) {
```

Comment:
Missing definitions for Easy, Normal, Hard, etc.

Reply:
Anything outside of Tower & Dan is assumed to be the default gamemode. Towers & Dans have exclusive animations which aren't loaded anywhere else.